TombEngine/Game/effects/item_fx.h

```diff
@@ -7,6 +8,10 @@ namespace TEN::Effects::Items
 	constexpr int DEFAULT_NONLETHAL_EFFECT_TIMEOUT = 48;
 
 	void ItemBurn(ItemInfo* item, int timeout = -1);
+	void ItemColorBurn(ItemInfo* item, ScriptColor const& color1 = ScriptColor(1, 1, 1), ScriptColor const& color2 = ScriptColor(1, 1, 1), int timeout = -1);
```

Comment:
Scripting code seems to follow this style, but everywhere else we've been doing `const Type&`. I'd be changing these to that format wherever you come across them along the way, just to be tidy. :)

Reply:
fixed on all of my codelines